test/form-layout.html

```diff
@@ -545,6 +545,24 @@
         expect(estimateEffectiveColspan(layout.children[0])).to.be.closeTo(2, .1);
       });
 
+      it('should update style after layout-item is not hidden', async() => {
+        sinon.spy(layout, '_invokeUpdateStyles');
+        const firstDisplay = layout.firstElementChild.style.display;
+        layout.firstElementChild.setAttribute('hidden', true);
```

Comment:
Boolean attributes should not have value. Rather just use the property.

Reply:
I changed to set property by `layout.firstElementChild.hidden = true;`